task5/task5.css

```diff
@@ -0,0 +1,115 @@
+body {
+    font-size: 100%;
+}
+
+.wrapper {
+    width: 90%;
+    margin: 0 auto;
+}
+
+.content-wrapper {
+    border: 2px solid black;
+    margin-top: 10px;
+}
+
+.content-wrapper .main-header {
+    text-align: center;
+}
+
+.content-wrapper .three-columns {
+    column-count: 3;
+    padding: 10px;
+}
+
+.wrapper .settings {
+    font-weight: bold;
+    margin: 10px;
+    margin-left: 30px;
+}
+
+.three-columns h2:first-of-type {
+    margin-top: 0;
+}
+
+.three-columns p {
+    text-align: justify;
+}
+
+input[type=radio] {
+    display: none;
+}
+
+.sans-serif,
+#sans-serif:checked~.content-wrapper {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.serif,
+#serif:checked~.content-wrapper {
+    font-family: 'Times New Roman', Georgia, Times, serif;
+}
+
+.small-font,
+#small-font:checked~.content-wrapper {
+    font-size: 0.85rem;
```

Comment:
just to be sure, do you understand that with pixels here you would have the same effect?

Reply:
Yes, except that using rems allows me to change only one line of css (root font-size) when I would like to change overall layout and everything would scale automatically